zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1936,51 +1940,80 @@ Available commands include:
     Returns "read_only".
 
 * *last_snapshot/lsnp* :
-    tbd
+    Information of the last snapshot that zookeeper server has finished saving to disk.
+    If called during the initial time period between the server starting up
+    and the server finishing saving its first snapshot, the command returns the
+    information of the snapshot read when starting up the server.
+    Returns "zxid" and "timestamp", the latter using a time unit of seconds.
 
 * *leader/lead* :
     If the ensemble is configured in quorum mode then emits the current leader
     status of the peer and the current leader location.
     Returns "is_leader", "leader_id", and "leader_ip".
 
 * *monitor/mntr* :
-    tbd
+    Emits a wide variety of useful info for monitoring.
+    Includes performance stats, information about internal queues, and
+    summaries of the data tree (among other things).
+    Returns each as its own field.
 
 * *observer_connection_stat_reset/orst* :
     Reset all observer connection statistics. Companion command to *observers*.
     No new fields returned.
 
 * *ruok* :
     No-op command, check if the server is running.
+    A response does not necessarily indicate that the
+    server has joined the quorum, just that the admin server
+    is active and bound to the specified port.
     No new fields returned.
 
 * *set_trace_mask/stmk* :
-    tbd
+    Sets the trace mask (as such, it requires a parameter).
+    Write version of *get_trace_mask*.
+    See the description of the four letter command *stmk* for
+    more details.
+    Returns "tracemask".
 
 * *server_stats/srvr* :
-    tbd
+    Server information.
+    Returns multiple fields giving a brief overview of server state.
 
 * *stats/stat* :
-    tbd
+    Same as *server_stats* but also returns "connections" field.
 
 * *stat_reset/srst* :
-    tbd
+    Resets server statistics. This is a subset of the information returned
+    by *server_stats* and *stats*.
+    No new fields returned.
 
 * *observers/obsr* :
-    tbd
+    Information on observer connections to server.
+    Always available on a Leader, available on a Follower if its
+    acting as a learner master.
+    Returns "synced_observers" (int) and "observers" (list of per-observer properties).
 
 * *system_properties/sysp* :
-    tbd
+    All defined system properties.
+    Returns each as its own field.
 
 * *voting_view* :
-    Print details about serving configuration.
+    Provides the current voting members in the ensemble.
+    Returns "current_config" as a map.
+
 * *watches/wchc* :
-    Print details about serving configuration.
+    Watch information aggregated by session.
```

Comment:
nitpick: Should we put a warning on the usage too?  `Note, depending on the
    number of watches this operation may be expensive (ie
    impact server performance), use it carefully.`

And for other commands too.

Reply:
Good idea, I'll do a pass over the commands and see which ones to annotate.